doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -609,7 +718,7 @@ and :math:`{z}_{bot}` (m), come from the AgroIBIS formulation:
 
 
 .. math::
-   :label: 25.10
+   :label: 25.16
```

Comment:
I can't seem to leave a comment where you haven't made changes, so sorry for the text mismatch here. I'm curious how fertilization rates for biofuels are selected. We have these on the land surface dataset by country and crop type. Did you add biofuels there? See the “Interactive Fertilization” section for details, and please update/adjust if necessary.